### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by callihiggins**

### CODE REVIEW

File: CoopANDlastfm.html [hunk 1]
```diff
@@ -7,25 +7,16 @@
 		<title>Sound and color with me</title>
 		<link rel="stylesheet" type="text/css" href="CoopANDlastfm.css"/>
 		<link rel="stylesheet" type="text/css" href="http://code.jquery.com/ui/1.9.2/themes/base/jquery-ui.css"/>
-
```
Comment (callihiggins) [inline CoopANDlastfm.html:10]: Put your scripts at the very bottom of body and css files in the head

File: CoopANDlastfm.html [hunk 2]
```diff
@@ -7,25 +7,16 @@
...
-		
+		<link rel="stylesheet" href="//ajax.googleapis.com/ajax/libs/jqueryui/1.11.3/themes/smoothness/jquery-ui.css">
 	</head>
 	<body>
 		<h2>SOUND AND COLOR</h2>
 		<h3> a project in synesthesia</h3>
 		<input id="query" type="text" placeholder="ENTER A COLOR"></input>
 		<button class="aButton" id="search">SUBMIT</button>
 		<div id="images"></div>
-		<div id="searchResults" div class="ui-widget-content"> </div>
```
Comment (callihiggins) [inline CoopANDlastfm.html:22]: The extra div in this element might have been a reason you were having problems on the reload

File: coopAndLastFM2.js [hunk 3]
```diff
@@ -95,11 +95,12 @@ var app = {
...
-				$("#searchResults").append(htmlString);
-				document.getElementById("searchResults").contentEditable='true';
-				$("#searchResults").draggable();
-    			$( "#searchResults").resizable();
+				// var htmlString = "<p class='lastFMresults'>" + searchResults + "</p>";
+				// $("#searchResults").append(htmlString);
+				// document.getElementById("searchResults").contentEditable='true';
+				// $("#searchResults").draggable();
+    // 			$( "#searchResults").resizable();
+    			songTitle = searchResults;
```
Comment (callihiggins) [inline coopAndLastFM2.js:103]: global variable for the songTitle that gets set to your results from lastFM

File: coopAndLastFM2.js [hunk 4]
```diff
@@ -112,7 +113,11 @@ var app = {
 		//options for user to change text in any way they would like 
 		$("#fonts").change(function() {
     	//alert($(this).val());
-   		 $('#searchResults').css("font-family", $(this).val());
+    	
+    		// set global font variable to select option
+    		font = $(this).val();
+
```
Comment (callihiggins) [inline coopAndLastFM2.js:119]: global variable for font that gets set when someone changes the font dropdown

File: coopAndLastFM2.js [hunk 5]
```diff
@@ -135,7 +140,9 @@ var app = {
 };
 
 // global variable to hold onto the image
-var theImage;
+
+//add variables for songTitle string and font string
+var theImage, font, songTitle;
```
Comment (callihiggins) [inline coopAndLastFM2.js]: global variables for the image, font, songtitle (and eventually font size? font color? whatever you want to change!)

File: coopAndLastFM2.js [hunk 6]
```diff
@@ -144,6 +151,9 @@ function setup() {
 
 	//Call the function to make the AJAX call
 	app.initialize();
+
+	// set an initial condition
+	font = "Arial";
 }
```
Comment (callihiggins) [inline coopAndLastFM2.js:160]: Initial condition for the font

File: coopAndLastFM2.js [hunk 7]
```diff
@@ -155,6 +165,12 @@ function draw() {
 		fill(0);
 		text("Getting Data");
 	}
+	
+	if(songTitle) {
+		textFont(font);
+		text(songTitle, width/2, height/2);
+	}
+	
 }
```
Comment (callihiggins) [inline coopAndLastFM2.js]: If we have songTitle data, draw the title using the font thats set on the variable

File: coopAndLastFM2.js [hunk 8]
```diff
@@ -144,6 +144,9 @@ var app = {
 //add variables for songTitle string and font string
 var theImage, font, songTitle;
 
+//green variabe for tint, unlocked boolean
+var g, unlocked = false;
+
```
Comment (callihiggins) [inline coopAndLastFM2.js]: variables for g value, and whether or not its locked

**Review by izzylee0 — APPROVED** [review]

**Review by izzylee0 — APPROVED** [review]

**Pull request merged**